algorithms/linfa-linear/Cargo.toml

```diff
@@ -17,16 +17,16 @@ keywords = ["machine-learning", "linfa", "ai", "ml", "linear"]
 categories = ["algorithms", "mathematics", "science"]
 
 [dependencies]
-ndarray = {version = "0.13", features = ["blas", "approx"]}
-ndarray-linalg = "0.12"
-ndarray-stats = "0.3"
+ndarray = {version = "0.14", features = ["blas", "approx"]}
```

Comment:
if you're correcting the spacing, do here as well. I actually don't know why rustfmt doesn't do that automatically

Reply:
done